src/CompositeCancellation.php

```diff
@@ -69,7 +69,8 @@ public function __destruct()
 
     public function subscribe(\Closure $callback): string
     {
-        $id = $this->nextId++;
+        $id = $this->nextId;
+        \PHP_VERSION_ID >= 80300 ? $this->nextId = \str_increment($this->nextId) : ++$this->nextId;
```

Comment:
```suggestion
        /** @psalm-suppress UndefinedFunction */
        \PHP_VERSION_ID >= 80300 ? $this->nextId = \str_increment($this->nextId) : ++$this->nextId;
```

Probably like this (as in revoltphp/event-loop#110)?

Reply:
I disabled it for that function globally instead.